src/app/packages/map-source-provider/map-source-provider.module.ts

```diff
@@ -0,0 +1,20 @@
+import { NgModule ,ModuleWithProviders , OpaqueToken} from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { ISourceProvider } from './models/models'
+import { sourceMapProviders,MapSourceProviderContainer } from './services/map-source-provider-container.service';
+
+@NgModule({
+    imports: [CommonModule],
+    exports: [MapSourceProviderContainer],
+    providers: [MapSourceProviderContainer],
```

Comment:
please fix MapSourceProviderContainer to MapSourceProviderContainerService
remove  exports